samples/cguibu.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupImageBasedUpgrade
+metadata:
+  name: upgrade-4.15.0
+  namespace: default
+spec:
+  clusterLabelSelectors:
+    - matchLabels:
+        common: "true"
+  ibuSpec:
+    seedImageRef:
+      image: quay.io/angwang/lca-seed-cnfdf20:4.15.0-rc.5-v4
```

Comment:
This can be change to something more upstream-ish? :)

Reply:
fixed it in the new PR.